tests/jballerina-unit-test/src/test/resources/test-src/annotations/deprecation_annotation_negative.bal

```diff
@@ -226,3 +226,34 @@ class Listener {
 service /s on new Listener() {
 
 }
+
+# The `Employee` is a user-defined record type
+#
+# + name - Employee name
+# + id - Employee id number
+# # Deprecated parameters       // warning
```

Comment:
Why does it say `'Deprecated parameters' documentation is not allowed here`? Is not this allowed in the spec?

Reply:
`name` is not a parameter. So, `# # Deprecated parameters` is not allowed here. It is used for when passing deprecated parameters to a method